### PR TITLE
fix oneccl CI/CD issue

### PR DESCRIPTION
**Pull request by louie-tsai**
# Existing Sample Changes
## Description

Fixed the sample.json issue 

Fixes Issue# ONSAM-1590

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

### CODE REVIEW

File: Libraries/oneCCL/tutorials/scripts/ci_test.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            with open(output_notebook_filename, 'w', encoding='utf-8') as f:
+                nbformat.write(nb, f)
+            return 0
+    except CellExecutionError:
+        print("Exception!")
+        return -1
+
+print(__file__)
+runJupyterNotebook(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                'oneCCL_Getting_Started.ipynb'),
```
Comment (e-zorina) [inline Libraries/oneCCL/tutorials/scripts/ci_test.py:25]: Please update the path of oneCCL_Getting_Started.ipynb here, it's one level up from this script, so this should be:
`os.path.join(os.path.dirname(os.path.realpath(__file__)), **'..'** ,'oneCCL_Getting_Started.ipynb'),  'oneCCL_Getting_Started_result.ipynb', 'base')`

Comment (louie-tsai) [reply Libraries/oneCCL/tutorials/scripts/ci_test.py:25]: @e-zorina 
We assume that CI/CD will run sample.json under oneCCL/tutorials folder, so we run ci_test.py "python scripts/ci_test.py"
https://github.com/intel-ai-tce/oneAPI-samples/blob/oneccl_ci/Libraries/oneCCL/tutorials/sample.json#L18

Therefore, we assume that we are in the same folder as oneCCL_Getting_Started.ipynb.

thanks

Comment (e-zorina) [reply Libraries/oneCCL/tutorials/scripts/ci_test.py:25]: Hi @louie-tsai, yes, CI runs this sample from another path. Also, you are using os.path.realpath(__file__) here, which is always a place where this script, ci_test.py, is located in. So, I don't think it works from any path.

Thanks,
Kate

Comment (louie-tsai) [reply Libraries/oneCCL/tutorials/scripts/ci_test.py:25]: changed it according to Zorina's feedback

Comment (jimmytwei) [reply Libraries/oneCCL/tutorials/scripts/ci_test.py:25]: @e-zorina @louie-tsai made the change. Just want to confirm this is the change that you requested.

**Pull request merged**